src/os_sensor_nodelet.cpp

```diff
@@ -449,8 +456,10 @@ class OusterSensor : public OusterClientBase {
    private:
     PacketMsg lidar_packet;
     PacketMsg imu_packet;
+    std_msgs::String config_msg;
```

Comment:
If you follow my advice where I suggest to move the `metadata_pub` then we don't need to have this member variable.
```suggestion
    std_msgs::String config_msg;
```